3-JSS/styles.js

```diff
@@ -48,10 +48,11 @@ const styles = {
     width: "100px",
   },
   buttonStyle : {
-    width: "40px",
-    height: "20px",
-    textAlign: 'center',
-    backgroundColor: 'green',
+    width: "10ch",
```

Comment:
don't use ch for buttons :) It was meant for the `input`.
You don't want your elements to be rigid and have constants for width and height. You want to use padding and the content itself to size the element.